app/models/work.rb

```diff
@@ -0,0 +1,23 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  validates :title, presence: true
+  validates :category, presence: true, inclusion: {in: %w(album book movie)}
+
+  def self.sort_work(category)
+    works = Work.where(category: category).to_a
+    works.sort_by! { |work| Vote.where(work_id: work.id).length }
```

Comment:
Just a note, it's more efficient to sort things using the database, using `order`